CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 20.10.2
+* Add a protocols section to `admin_peers` to provide info about peer health. [\#1562](https://github.com/hyperledger/besu/pull/1562)
```

Comment:
I think you mean 1582

Reply:
and should this be under the Additions and Improvements heading?

Reply:
Done.